server/routes/api.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express')
+const router = express.Router()
+const moment = require('moment')
+const Expense = require('../models/Expense')
+
+router.get('/expenses', function (req, res) {
+    Expense.find({}, function (err, expense) {
+        res.send(expense)
+    }).sort({ date: -1 })
+})
+
+router.post('/new', function (req, res) {
+    let { name, amount, group, date } = req.body;
+    if (isNaN(date)) {
+        date = moment().format('LLLL')
+    }
+    const eToAdd = new Expense({ name, amount, group, date })
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: expenceToAdd etc.) and not something general like "arr", "str" etc.